CHANGELOG.md

```diff
@@ -131,6 +131,65 @@ In addition, these existing libraries are now GA:
+- feat(pubsub): implement per-call options for `Subscriber` ([#10043](https://github.com/googleapis/google-cloud-cpp/pull/10043))
+- fix(pubsub): limit `ModifyAckDeadlineRequest` size ([#10032](https://github.com/googleapis/google-cloud-cpp/pull/10032))
+- fix(pubsub): race condition capturing batching timer ([#9996](https://github.com/googleapis/google-cloud-cpp/pull/9996))
+- fix(pubsub): faster shutdowns for `Publisher` ([#9991](https://github.com/googleapis/google-cloud-cpp/pull/9991))
+
+### [Spanner](/google/cloud/spanner/README.md)
+
+- doc(spanner): add `*Client` samples ([#10145](https://github.com/googleapis/google-cloud-cpp/pull/10145))
+- feat(spanner): add support for RowStream::RowsModified() ([#10102](https://github.com/googleapis/google-cloud-cpp/pull/10102))
+- feat(spanner): support for the PG.JSONB data type ([#10098](https://github.com/googleapis/google-cloud-cpp/pull/10098))
+
+### [Storage](/google/cloud/storage/README.md)
+
+- doc(storage): common initialization examples ([#10107](https://github.com/googleapis/google-cloud-cpp/pull/10107))
+- fix(rest): too many debug headers ([#10054](https://github.com/googleapis/google-cloud-cpp/pull/10054))
+- fix(rest): return complete payloads for errors ([#10051](https://github.com/googleapis/google-cloud-cpp/pull/10051))
+- feat(storage): support `Autoclass` feature ([#10003](https://github.com/googleapis/google-cloud-cpp/pull/10003))
+- feat(storage): faster `InsertObject()` uploads ([#9997](https://github.com/googleapis/google-cloud-cpp/pull/9997))
+- feat(GCS+gRPC): use alternative names in ACLs ([#9992](https://github.com/googleapis/google-cloud-cpp/pull/9992))
+- fix(storage): respect MIME message boundary size limits ([#9965](https://github.com/googleapis/google-cloud-cpp/pull/9965))
+
+### [Common Libraries](/google/cloud/README.md)
+
+- feat(generator): scaffold supports auth samples ([#10146](https://github.com/googleapis/google-cloud-cpp/pull/10146))
+- feat(generator): generate authentication example ([#10138](https://github.com/googleapis/google-cloud-cpp/pull/10138))
+- doc: workaround Doxygen formatting quirk ([#10137](https://github.com/googleapis/google-cloud-cpp/pull/10137))
+- doc: add endpoint override snippets to generated libs ([#10129](https://github.com/googleapis/google-cloud-cpp/pull/10129))
+- feat(generator): add markers during scaffold generation ([#10127](https://github.com/googleapis/google-cloud-cpp/pull/10127))
+- feat(generator): includes samples in DOXYGEN_EXAMPLE_PATH ([#10126](https://github.com/googleapis/google-cloud-cpp/pull/10126))
+- feat(generator): build scripts for samples ([#10125](https://github.com/googleapis/google-cloud-cpp/pull/10125))
+- feat(generator): generate simple samples for `*Client` ([#10118](https://github.com/googleapis/google-cloud-cpp/pull/10118))
+- doc: mocks library / StreamRange ([#10122](https://github.com/googleapis/google-cloud-cpp/pull/10122))
```

Comment:
nit: I think we can delete this one, because it is covered by the change below.

Reply:
Removed.